README.md

```diff
@@ -1,2 +1,29 @@
 # comp110-worksheet-B
-Base repository for COMP110 worksheet B
+
+
+## Pseudocode:
+
+```
+Create RWord <- a random word with x number of letters
+Create Tries <- 0 (current number of user attempts)
+
+Display list of random words, including RWord
+
+While Tries is <5:
+	Allow user to select word from list
+	If user's word is the same as RWord:
+		Display "success! Well done!" and end program
+	Else:
+		Add 1 to Tries and calculate identical letters 
```

Comment:
How are identical letters calculated? This is the algorithm that I wanted to see pseudocode for!